contrib/hadoop-store-builder/src/java/voldemort/store/readonly/fetcher/BasicFetchStrategy.java

```diff
@@ -89,82 +84,86 @@ private CheckSum copyFileWithCheckSum(HdfsFile source, File dest, CheckSumType c
         OutputStream output = null;
         long startTimeMS = System.currentTimeMillis();
 
-        for (int attempt = 0; attempt < fetcher.getMaxAttempts(); attempt++) {
```

Comment:
Sorry for the trouble caused here, from time to time my eclipse dies and I have to start the editor formatting from the first. It happens once every 6 months or so, this happened recently to my macbook and hence the code I wrote from macbook are mostly busted. Now I have fixed the formatting.